File: clientside/src/app/entities/potrosac.ts
import { Lokacija } from "./lokacija";

export class Potrosac {
    id:number;
    ime:String;
    prezime:String;
    ulica:String;
    grad:String;
    broj_telefona:String;
    tip:String;
    prioritet:number;
    postanski_broj:String;
    constructor(ime:String,prezime:String,ulica:String,grad:String,tip:String,broj_telefona? :String,postanski_broj?:String,prioritet?:number)
    {
        this.ime=ime;
        this.prezime=prezime;
        this.ulica=ulica;
        this.grad=grad;
        this.broj_telefona=broj_telefona;
        this.tip=tip;
        this.prioritet=prioritet;
        this.postanski_broj=postanski_broj;
        
    }

}

File: Core/SmartGrid/Models/Potrosac.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace SmartGrid.Models
{
    public class Potrosac
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        [Required]
        public string Ime { get; set; }
        [Required]
        public string Prezime { get; set; }
        [Required]
        public string Ulica { get; set; }
        [Required]
        public string Grad { get; set; }
        [Required]
        public int Broj { get; set; }


        public string Broj_telefona { get; set; }

        [Required]
        public string Tip { get; set; }
        public int Prioritet { get; set; }
        
        public int Postanski_broj { get; set; }
    }
}
